nemo/collections/asr/parts/k2/topologies.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+import k2
+
+
+def build_topo(name: str, tokens: List[int], with_selfloops: bool = True) -> k2.Fsa:
```

Comment:
Return type 'k2.Fsa'

Reply:
Done in [7b2b4a4](https://github.com/NVIDIA/NeMo/pull/3351/commits/7b2b4a4ba4e34848d526d8c4ac53903bfbfe2fd2)